proxy/proxy.go

```diff
@@ -237,7 +245,7 @@ type Params struct {
 
 	// OpenTracing contains parameters related to OpenTracing instrumentation. For default values
 	// check OpenTracingParams
-	OpenTracing *OpenTracingParams
+	OpenTracing *TracingParams
```

Comment:
I'm not very happy with the name of this parameter. But I think it's a bit weird to have two `TracingParams`, one for OT and other for OTel. I'm open to suggestions.

Reply:
We need to have a backwards compatible change, so better not to change the original type names.